contracts/DomainUtilsAssembly.sol

```diff
@@ -0,0 +1,78 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.20;
+
+library DomainUtilsAssembly {
+    
+    function stripProtocol(string memory domain) internal pure returns (string memory) {
+        bytes memory httpsPrefix = "https://";
+        bytes memory httpPrefix = "http://";
```

Comment:
Why not to strip everything before and including `// `?

Reply:
improved